docs/sce_file_contents.md

```diff
@@ -33,7 +34,9 @@ The `counts` assay can be accessed with the following R code:
 counts(sce) # counts matrix
 ```
 
-### SingleCellExperiment cell metrics
+Additionally, the `spliced` assay contains a counts matrix that includes reads from spliced cDNA only.
+
+### Cell metrics
```

Comment:
```suggestion
### SingleCellExperiment Cell metrics
```
We actually want this to say `SingleCellExperiment`. That allows us to separate out `AnnData` and `SingleCellExperiment` instructions and is consistent with the rest of this document.